back/src/v1/hosts.ts

```diff
@@ -144,7 +146,19 @@ router.get('/', async (req, res) => {
     delete host.latitude;
   });
 
-  sendResponse(res, formattedData, 200);
+  const elementsPerPage = 20;
+
+  const totalPages = Math.ceil(data.length / elementsPerPage);
+  const isLastPage = body.data.page === totalPages;
+
+  const startIndex = (body.data.page - 1) * elementsPerPage;
+  const endIndex = startIndex + elementsPerPage;
+
+  const currentPageData = formattedData.slice(startIndex, endIndex);
+
+  const returnedData = { isLastPage: isLastPage, hosts: currentPageData };
+
+  sendResponse(res, returnedData, 200);
```

Comment:
You should make this an utils method to reduce code duplication with the artist

If I understand the code correctly you fetch the all the artist every time I make a request ? Now that we have a pagination you should use the "limit" of prisma